TokenCacheMigration/CommonCacheAdalV5/FilesBasedTokenCache.cs

```diff
@@ -0,0 +1,119 @@
+﻿/*
+ The MIT License (MIT)
+
+Copyright (c) 2015 Microsoft Corporation
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+using Microsoft.Identity.Core.Cache;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using System.IO;
+using System.Security.Cryptography;
+
+namespace CommonCacheADALV5
+{
+ /// <summary>
+ /// Simple file based persistent cache implementation for a desktop application (from ADAL 4.x)
+ /// </summary>
+ class FilesBasedTokenCache : TokenCache
```

Comment:
Spacing looks a bit off.

Reply:
yes. thanks for catching this. I used a 1 space indent to be able to have code that is not too wide for the docs / wikis. Will fix this